src/pages/Home.vue

```diff
@@ -100,6 +122,10 @@ export default {
     this.tableData = this.dummyTableData;
     if (this.isUserApproved) await this.fetchPlioIds();
     this.$mixpanel.track("Visit Home");
+
+    // show the chatwoot bubble
+    var chatwootBubble = document.querySelector(".woot-widget-bubble");
+    if (chatwootBubble != undefined) chatwootBubble.classList.remove("hidden");
```

Comment:
this should not be needed anymore right? is it to handle navigating the app pages - that should be done by looking for the change of `onHomePage` value on App.vue right?